Example/ViewController.swift

```diff
@@ -558,7 +558,7 @@ extension ViewController: NavigationMapViewDelegate {
         return lineLayer
     }
     
-    func navigationMapView(_ navigationMapView: NavigationMapView, shapeFor route: Route) -> LineString? {
+    func navigationMapView(_ navigationMapView: NavigationMapView, shapeFor routes: [Route]) -> LineString? {
```

Comment:
We do not support one shape for all routes since `1.x`. I think this delegate method should be called multiple times per each route, so this change will have to reverted.

Reply:
I see. It should be similar to `func addRouteLayer`, should be called multiple times, every time form one shape for one route. Thanks for the suggestion, I'll fix this part.